proposals/lip-0030.md

```diff
@@ -53,10 +54,11 @@ Accounts are serialized and deserialized according to the `account` schema. The
 1. `address`: The account address. This can either be:
     1. The new address format, used after the implementation of [LIP 0018](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0018.md). This value is 20 bytes long.
     2. The legacy address, used before the implementation of [LIP 0018](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0018.md). This value is 8 bytes long.
-2. `balance`: The account balance in Beddows.
-3. `keys`: Accounts that issued a Multisignature Transaction store the transaction-signing rules in this property.
-4. <code>[nonce](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0015.md#nonce)</code>: The current nonce, representing the total number of transactions sent from the account.
-5. <code>asset</code>: The account asset stores extra information related to transaction assets. In this LIP we specify the asset schema for the Lisk mainchain.
+2. `token`: This property corresponds to the token module and contains the account balance in Beddows.
+3. `sequence`: This property corresponds to the sequence module and contains the current [nonce](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0015.md#nonce), representing the total number of transactions sent from the account.
+4. `keys`: Accounts that issued a Multisignature Transaction store the transaction-signing rules in this property.
```

Comment:
I think now the preferred way for transaction types is to write everything lowercase.